Update devGuide for translating symbols, gestures, character descriptions (#17590)
nvaccess/nvda: projectDocs/dev/developerGuide/developerGuide.md

> #### Translating this file {#TranslatingSymbolsFile}

Reviewer: I'd move this paragraph at the end. Indeed the sentence before announces the two next paragraphs.

#### Translating this file {#TranslatingGesturesFile}

Translation of `gestures.ini` happens via [Pull Request to NVDA](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/contributing.md).

> 1. In your local copy of the screenReaderTranslations repository, check if the `gestures.ini` file exists, e.g. `d:\SVN\SRT\fr\gestures.ini`

Reviewer: Update SVN to Git:

```suggestion
1. In your local copy of the NVDA repository, check if the `gestures.ini` file exists, e.g. `nvda\source\locale\fr\gestures.ini`
```

Reviewer: And also a bit more below, remove this line:
`6. Commit your changes to the screenReaderTranslations repo.`

> 1. In your local copy of the screenReaderTranslations repository, check if the `gestures.ini` file exists, e.g. `d:\SVN\SRT\fr\gestures.ini`

Reviewer 2: This still refers to SVN.

>    * If this file does not exist, create it by copying it from the last version of NVDA.

Reviewer: ```suggestion
   * If this file does not exist, create it by copying and adapting it from another language.
```
